eng/testing/tests.props

```diff
@@ -6,6 +6,7 @@
     <TestDependsOn Condition="'$(TestNoBuild)' != 'true' and '$(BuildAllProjects)' != 'true'">Build</TestDependsOn>
     <TestDependsOn>$(TestDependsOn);GenerateRunScript;RunTests</TestDependsOn>
     <VSTestNoLogo>true</VSTestNoLogo>
+    <ILDescriptorsPath>$(MSBuildThisFileDirectory)ILDescriptors\</ILDescriptorsPath>
```

Comment:
I think this should be named `ILLinkDescriptors`. Both the folder name and the MSBuild properties.